### PR TITLE
Deploy staging version of docs to staging bucket

### PR DESCRIPTION
**Pull request by travis**
In preparation for a real staging env for the static site.

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -13,4 +14,5 @@ deployment:
   prod:
     branch: master
     commands:
+      - aws s3 sync jekyll/_staging_site s3://circle-staging-docs --delete
```
Comment (z00b) [inline circle.yml]: as i mentioned in the other pr, i'd use a bucket that is itselft a viable hostname, then you can just use static hosting.

Comment (travis) [inline circle.yml]: see other post - we need multiple buckets stitched together

**Comment by ndintenfass** [comment]
Lgtm

**Comment by z00b** [comment]
:+1:

**Pull request merged**